samwise.js

```diff
@@ -2,107 +2,111 @@
  * Autoplay plugin - Automatically advance slideshow after N seconds
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?